src/sidebar/store/modules/test/filters-test.js

```diff
@@ -45,6 +45,15 @@ describe('sidebar/store/modules/filters', () => {
         assert.isFalse(filterState.focusActive);
         assert.isUndefined(filterState.focusFilters.user);
       });
+
+      it('un-sets and deactivates the user focus filter when null user value is provided', () => {
+        store.toggleFocusMode(true);
+        store.changeFocusModeUser({ user: null });
```

Comment:
To make this test more realistic it might be useful to first set an actual user and then clear it.